src/pages/Admin/Venue/Rooms/RoomsValidationSchema.ts

```diff
@@ -26,7 +27,9 @@ export const validationSchema = Yup.object()
     title: Yup.string().required("Required"),
     subtitle: Yup.string().required("Required"),
     about: Yup.string().required("Required"),
-    url: Yup.string().required("Required"),
+    url: Yup.string()
+      .required("Required")
+      .test("url validation", "Enter correct url", isValidUrl),
```

Comment:
Can we export `roomUrlSchema` from `src/pages/Admin/Details/ValidationSchema.ts` and then reuse it directly here?

```suggestion
    url: roomUrlSchema,
```

Reply:
fixed